lib/note-toolbar/index.tsx

```diff
@@ -184,4 +188,8 @@ const mapStateToProps: S.MapState<StateProps> = ({ ui: { note } }) => ({
   note,
 });
 
-export default connect(mapStateToProps)(NoteToolbar);
+const mapDispatchToProps = (dispatch, {}) => ({
+  onToggleNoteInfo: () => dispatch(toggleNoteInfo(true)),
+});
```

Comment:
if we remove the `show` parameter as discussed in the `reducer` file below then this simplifies

```ts
const mapDispatchToProps: S.MapDispatch<DispatchProps> = {
  onToggleNoteInfo: toggleNoteInfo,
}
```